apps/condo/domains/miniapp/utils/testSchema/index.js

```diff
@@ -32,23 +32,25 @@ const { B2BAppPermission: B2BAppPermissionGQL } = require('@condo/domains/miniap
 const { B2BAppRole: B2BAppRoleGQL } = require('@condo/domains/miniapp/gql')
 const { B2BAppAccessRightSet: B2BAppAccessRightSetGQL } = require('@condo/domains/miniapp/gql')
 const { B2BAppNewsSharingConfig: B2BAppNewsSharingConfigGQL } = require('@condo/domains/miniapp/gql')
-const { B2CAppMessageSetting: B2CAppMessageSettingGQL } = require('@condo/domains/miniapp/gql')
 const { B2BAccessToken: B2BAccessTokenGQL } = require('@condo/domains/miniapp/gql')
 const { B2BAccessTokenAdmin: B2BAccessTokenAdminGQL } = require('@condo/domains/miniapp/gql')
 const { B2BAccessTokenReadonly: B2BAccessTokenReadonlyGQL } = require('@condo/domains/miniapp/gql')
 const { B2BAccessTokenReadonlyAdmin: B2BAccessTokenReadonlyAdminGQL } = require('@condo/domains/miniapp/gql')
+const { AppMessageSetting: AppMessageSettingGQL } = require('@condo/domains/miniapp/gql')
+const { SEND_B2B_APP_PUSH_MESSAGE_MUTATION } = require('@condo/domains/miniapp/gql')
+
 /* AUTOGENERATE MARKER <IMPORT> */
 
-function randomChoice(options) {
-    const index = Math.floor(Math.random() * options.length);
-    return options[index];
+function randomChoice (options) {
+    const index = Math.floor(Math.random() * options.length)
+    return options[index]
```

Comment:
You can use `faker.helpers.arrayElement([a, b, c]`, it is return `a` or `b` or `c`.
But ok, its some legacy